ivy/data_classes/array/array.py

```diff
@@ -908,12 +908,20 @@ def __gt__(self, other):
         >>> z = x > y
         >>> print(z)
         {
-            a: ivy.array([[True, False, False],
-                          [False, False, False],
-                          [False, False, False]]),
-            b: ivy.array([[True, True, False],
-                          [False, False, False],
-                          [False, False, False]])
+            a: ivy.array(
```

Comment:
I think the array above was originally formatted correctly, given that all frameworks format it that way. Same applies to other similar instances.